devZoGok/Battleship
# Ft 36 units

Implemented basic set of unit functionality.

Test steps for garrisoning:
1) create a transport via `add-unit <player_id> <unit_id>` where `player_id` is 1 and `unit_id` is 2
2) create another non transport vehicle for the player themself
3) select the non transport vehicle
4) left click the transport
5) observe the order line become blue
6) select the transport
7) observe its garrison slot to turn green

Test steps for attacking:
1) create a war mech via `add-unit <player_id> <unit_id>` where `player_id` is 1 and `unit_id` is 0
2) create another unit
3) select the war mech
4) left click the other unit while holding Left Control
5) observe the order line become red and the war mech approach its target if it's far apart enough
6) observe the target disappear with an explosion SFX
